src/Service/Lambda/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
 
 - AWS api-change: Restrict the list of available regions.
 - AWS api-change: Added `eu-central-2` region
+- AWS api-change: Added support for customization of Consumer Group ID for MSK and Kafka Event Source Mappings.
```

Comment:
None of these change log entries are relevant, and we're missing a note for the new eu region.